lib/shopify_transporter/pipeline/magento/product/variant_image.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+require 'shopify_transporter/pipeline/stage'
+require 'shopify_transporter/shopify'
+
+module ShopifyTransporter
+  module Pipeline
+    module Magento
+      module Product
+        class VariantImage < Pipeline::Stage
+          def convert(hash, record)
+            return unless input_applied?(hash)
+            accumulate(hash, record)
+            add_variant_img_to_parent_image_array(record, hash)
```

Comment:
The order of parameters should be consistent -- you have `hash, record` for the `convert` method, and then `record, hash` for the `add_variant_img_to_parent_image_array` method.